be/src/http/action/compaction_action.cpp

```diff
@@ -275,4 +282,20 @@ void CompactionAction::handle(HttpRequest* req) {
         }
     }
 }
+
+std::shared_ptr<CumulativeCompactionPolicy> CompactionAction::_update_cumulative_compaction_policy() {
```

Comment:
The method name is confused. Actually it is "check-and-create", not "update".

Reply:
> 

OK, Thank you.